exercises/error-handling/src/test/java/ErrorHandlingTest.java

```diff
@@ -0,0 +1,152 @@
+import junit.framework.AssertionFailedError;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+
+public class ErrorHandlingTest {
+
+    private ErrorHandling errorHandling;
+    private Set<Class> checkedExceptions;
```

Comment:
Why do you need these two sets? You've already provided the starter implementation for the exception you want them to throw (`CustomCheckedException` and `CustomUncheckedException`), so you know which class it should be 🙂

Reply:
I added them before providing the implementation and I forgot them 😅 Sets removed 🙂